src/onegov/core/layout.py

```diff
@@ -154,40 +151,8 @@ def csrf_protected_url(self, url: str) -> str:
         return self.request.csrf_protected_url(url)
 
     def format_date(self, dt: datetime | date | None, format: str) -> str:
-        """ Takes a datetime and formats it according to local timezone and
-        the given format.
-
-        """
-        if dt is None:
-            return ''
-
-        if getattr(dt, 'tzinfo', None) is not None:
-            dt = self.timezone.normalize(
-                dt.astimezone(self.timezone)  # type:ignore[attr-defined]
-            )
-
-        locale = self.request.locale
-        assert locale is not None, 'Cannot format date without a locale'
-        if format == 'relative':
-            adt = arrow.get(dt)
-
-            try:
-                return adt.humanize(locale=locale)
-            except ValueError:
-                return adt.humanize(locale=locale.split('_')[0])
-
-        fmt = getattr(self, format + '_format')
-        if fmt.startswith('skeleton:'):
-            return babel.dates.format_skeleton(
-                fmt.replace('skeleton:', ''),
-                datetime=dt,
-                fuzzy=False,
-                locale=locale
-            )
-        elif hasattr(dt, 'hour'):
-            return babel.dates.format_datetime(dt, format=fmt, locale=locale)
-        else:
-            return babel.dates.format_date(dt, format=fmt, locale=locale)
+        fmt = getattr(self, f'{format}_format', format)
```

Comment:
I would move this part to `CoreRequest` as well, we should never manually specify a `fmt` string, it should always be one of our pre-configured format options. If one of the applications overwrites the formats in their specific layout class, you would need to instead overwrite them in their specific request class, but I'm not sure if we do overwrite them anywhere.